db/db_test2.cc

```diff
@@ -2882,6 +2882,79 @@ TEST_F(DBTest2, TraceWithLimit) {
   ASSERT_OK(DestroyDB(dbname2, options));
 }
 
+TEST_F(DBTest2, TraceWithSampling) {
+  Options options = CurrentOptions();
+  options.merge_operator = MergeOperators::CreatePutOperator();
```

Comment:
You can remove this merge_operator here and on line 2926 as you don't have any Merge API calls.